data/json/items/chemicals_and_resources.json

```diff
@@ -833,6 +833,54 @@
     "stack_size" : 100
    },
    {
+    "id": "compost_unf",
+    "type": "AMMO",
+    "category" : "spare_parts",
+    "symbol": "%",
+    "color": "black",
+    "name": "compost",
+    "description": "A reeking mush of rotting vegetation and soil. Activate to start composting it!",
+    "price": 0,
+    "material": "powder",
+    "flags": ["NO_SALVAGE"],
+	"ammo_type" : "components",
+    "weight": 400,
+    "volume": 1,
+	"charges" : 5,	
+    "to_hit": -2,
+    "use_action": {
+        "type": "delayed_transform",
```

Comment:
Delayed transform is a very annoying mechanic to use.
Considering that `"transform_age"` of 1 day is nearly trivial and that it doesn't occupy any container, it would be better to just skip it and craft the finished version.